src/goto-instrument/wmm/goto2graph.cpp

```diff
@@ -145,14 +145,11 @@ unsigned instrumentert::goto2graph_cfg(
 }
 
 void instrumentert::cfg_visitort::visit_cfg_function(
-    /* value_sets and options */
```

Comment:
Why do these comments need to be removed? Surely Doxygen should just be ignoring these?

Reply:
Ah, apologies - I see they are replaced by Doxygen on the declaration instead. Ignore me :-)